client/extensions/woocommerce/style.scss

```diff
@@ -1,10 +1,64 @@
+.is-section-woocommerce {
+	.layout__primary {
+		display: flex;
+		justify-content: center;
+	}
+}
+
 .woocommerce {
+	flex-grow: 1;
+	max-width: 720px;
```

Comment:
Can we put these rules under the class `.products__form`? The max width may be over reaching and need to be overwritten for something like the stats page.